README.md

```diff
@@ -165,14 +165,15 @@ project.pbxproj.objects.addObject(homeGroup)
 
 ```swift
 let homeGroup = PBXGroup(reference: "xxx", children: [], sourceTree: .group, path: "Home")
-let homeViewController = PBXFileReference(reference: "xxx", sourceTree: .group, path: "HomeViewController.swift")
+let homeViewController = PBXFileReference(reference: project.pbxproj.generateUUID(for: PBXFileReference.self), sourceTree: .group, name: "HomeViewController.swift", path: "HomeViewController.swift")
 homeGroup.children.append(homeViewController.reference)
+pbxproj.objects.addObject(homeViewController)
 ```
 
 #### Add `HomeViewController.swift` file to `MyApp` target
 
 ```swift
-let homeViewController = PBXFileReference(reference: "xxx", sourceTree: .group, path: "HomeViewController.swift")
+let homeViewController = PBXFileReference(reference: project.pbxproj.generateUUID(for: PBXFileReference.self), sourceTree: .group, name: "HomeViewController.swift", path: "HomeViewController.swift")
```

Comment:
Name and path of the file may be different, i.e. when file is added with absolute path which is not in a group's folder